tests/runner.py

```diff
@@ -2019,10 +2024,12 @@ def flattened_tests(loaded_tests):
 
 def suite_for_module(module, tests):
   suite_supported = module.__name__ in ('test_core', 'test_other')
-  has_multiple_tests = len(tests) > 1
-  has_multiple_cores = parallel_testsuite.num_cores() > 1
-  if suite_supported and has_multiple_tests and has_multiple_cores:
-    return parallel_testsuite.ParallelTestSuite(len(tests))
+  if not EMTEST_SAVE_DIR:
+    has_multiple_tests = len(tests) > 1
+    has_multiple_cores = parallel_testsuite.num_cores() > 1
+    if suite_supported and has_multiple_tests and has_multiple_cores:
+      return parallel_testsuite.ParallelTestSuite(len(tests))
+  print('Using single-threads tested processing')
```

Comment:
maybe `Using single-threaded test processing` ?

but also, should this be `print`ed all the time? I'm not opposed, just want to make sure it's not a mistake.

Reply:
Yes, since the parallel test runner unconditionally prints this.. but you are right maybe its not needed.